CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - [case: PBLD-6] Fixed unnecessary reimport of all project textures on package install.
 - [case: PBLD-11] Fixed Poly Shape Tool missing an undo step after setting height.
 - [case: PBLD-13] Fixed a bug where a newly created shape would not be redrawn with Redo. 
+- [case: PBLD-15] Fixed URP support in the Game View.
```

Comment:
@JonMUnity Thanks for the review! The issue was that the render pipeline was not properly set to URP in the `Runtime/Core/SelectionPickerRendererStandard.cs`. We were previously only modifying the render pipeline in the `GraphicsSettings`, but we also needed to do the same in the `QualitySettings`. The bug stated that it wasn't possible to pick faces in the Game View, but since the render pipeline was not properly set, it makes me think that it might be more widespread.

That being said, I think the updated log message is very appropriate for the fix made. :-)

Reply:
Great! Thanks for the reply, Claudia!